src/platform/testing/unit/mocha-setup.js

```diff
@@ -30,8 +36,21 @@ export default function setupJSDom() {
 
   // Prevent warnings from displaying
   /* eslint-disable no-console */
-  console.error = () => {};
-  console.warn = () => {};
+  if (process.env.LOG_LEVEL === 'debug') {
+    console.error = (error, reactError) => {
+      if (reactError instanceof Error) {
```

Comment:
The React 16 error logging is weird with error boundaries. It logs out the original error, along with a different, higher level error, which doesn't contain any useful info. This way, we log the real error in a React component and skip all the other extraneous info.

Reply:
nice, the `env.LOG_LEVEL` approach was my first thought, handling the error more thoroughly is even better